app/src/main/java/org/mozilla/fenix/compose/cfr/CFRPopup.kt

```diff
@@ -106,9 +106,9 @@ class CFRPopup(
 
         /**
          * How tall the indicator arrow should be.
-         * This will also affect how wide the base of the indicator arrow will be.
+         * This will also the width of the indicator's base which is double the value.
```

Comment:
nit: "also set the width" or something similar.

Reply:
Thank you. Definitely deleted more text that I intended.
Will change to "This will also affect the width of the indicator's base which is double the height value"